prow/plugins.yaml

```diff
@@ -599,6 +600,17 @@ plugins:
   - size
   - welcome
 
+  GoogleCloudPlatform/netd:
```

Comment:
No cat?!

Reply:
:100: please consider add our best selling prow plugins:  `- cat` and `- dog`!

Reply:
The cat plugin adds a cat image to an issue in response to the `/meow` command.
Aha, I love it.